qiskit_experiments/test/Fine Calibration Tutorial.rst

```diff
@@ -0,0 +1,154 @@
+================================================
+Fine Calibrations with Pulse Simulator
+================================================
+The amplitude of a pulse can be precisely calibrated using
+error amplifying gate sequences. These gate sequences apply 
+the same gate a variable number of times. Therefore, if each gate
+has a small error dtheta  in the rotation angle then 
+a sequence of n gates will have a rotation error of n*dtheta.
```

Comment:
Can you use :math:? I think it should render properly when you build the docs.
```suggestion
has a small error :math:`{\rm d}\theta`  in the rotation angle then 
a sequence of :math:`n` gates will have a rotation error of :math:`n\cdot {\rm d}\theta`.
```

Reply:
done